src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -812,6 +815,13 @@ export async function install(
   try {
     const gameInfo = getGameInfo(appName)
     const { title } = gameInfo
+    if (gameInfo.type === 'mod' && modOptions?.zipFilePath) {
```

Comment:
this should be if game is market wars for now since it is not general to all mods